### DISCUSSION TITLE
Suggest Opening all chests for pots and gold in early game ( bodys, barrels and so on, especially in Den of Evil and first two maps ))

### DISCUSSION THREAD

**Discussion by tyson-dev** [isid0re/SoloLeveling#329] [General]
I Am not sure if posting yt links is fine but if ya search for like Kolbot Solutions: Open All Chests (Not ChestMania) you'll get the point.

Edit: Actually adding the rogue bodys should be enough, he is doing the hidden stones and chests already, i keep watching.

**Comment by isid0re** [comment]
I actually have that done to open everything in hell only. As it does slow down the runtime if it pops everything. Ive tried to find a balance between the 2, but I can start a timer and if the time addition isnt much than it might be worth while to add for early gold.

I'll check into it. Thanks

**Reply by tyson-dev** [reply]
Yeah going over the top with this can really get annoying e.g.: with barrels all over the place. Maybe if we just add the "roguecorpse" it should be fine, also your timersolution sounds good as it gets obsolete later on anyway.
